compiler/noirc_frontend/src/monomorphization/mod.rs

```diff
@@ -458,11 +474,18 @@ impl<'interner> Monomorphizer<'interner> {
                     // If an impl was selected for this infix operator, replace it
                     // with a method call to the appropriate trait impl method.
                     let (function_type, ret) =
-                        self.interner.get_operator_type(infix.lhs, operator, expr);
+                        self.interner.get_infix_operator_type(infix.lhs, operator, expr);
 
                     let method = infix.trait_method_id;
                     let func = self.resolve_trait_method_expr(expr, function_type, method)?;
-                    self.create_operator_impl_call(func, lhs, infix.operator, rhs, ret, location)?
+                    self.create_infix_operator_impl_call(
+                        func,
+                        lhs,
+                        infix.operator,
```

Comment:
If you create a let binding for `infix.operator` does that avoid splitting this call onto multiple lines?

Reply:
This hasn't been changed, so I assume the answer was no?

Reply:
Oh, I thought I did it but apparently there were two such requests: https://github.com/noir-lang/noir/pull/5423/commits/3c8c155696c411568868da2ed65fcb729aa0e012

Reply:
Or maybe I did it in the wrong place 😅